--- data/version_1.py	2020-03-26 04:21:50.000000000 -0500
+++ data/version_2.py	2020-03-26 00:05:16.000000000 -0500
@@ -4,2 +4 @@
-    supported_op = ['add']
-    # command = "add 20 and 3"
+    print("Given a math operation command, the program parses and performs it")
@@ -7 +6,2 @@
-    _len = len(command)
+    supported_op=['add','subtract','multiply','subtract','divide']
+    # command="divide 20 and 10"
@@ -41 +41 @@
-    result = 0
+    result=0
@@ -43,2 +43,2 @@
-    if flag == False:
-        print("Invalid command")
+    if flag==False:
+        print(op+" operation not supported")
@@ -46,8 +46,2 @@
-        if op == 'add':
-            result = num1 + num2
-
-        print(op, num1, ' and ', num2, '= ', result)
-
-
-if __name__ == "__main__":
-    process(sys.argv[1])
+        if op=='add':
+            result=num1+num2
@@ -54,0 +49,2 @@
+        if op=='subtract':
+            result=num1-num2
@@ -55,0 +52,2 @@
+        if op=='multiply':
+            result=num1*num2
@@ -56,0 +55,2 @@
+        if op=='divide':
+            result=num1/num2
@@ -57,0 +58,2 @@
+        #result casted to int
+        result=int(result)
@@ -58,0 +61 @@
+        print(op,num1,' and ',num2,'= ',result)
@@ -59,0 +63,2 @@
+if __name__ == "__main__":
+    process(sys.argv[1])
